### PR TITLE
New package: kamoso-21.04.3_1

### PR DESCRIPTION
**Pull request by hervyqa**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

#### Does it build and run successfully? 
(Please choose at least one native build and, if supported, at least one cross build. More are better.)
- [x] I built this PR locally for my native architecture, (x86_64)
- [ ] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [ ] aarch64-musl
  - [ ] armv7l
  - [ ] armv6l-musl

Screenshot_20210714_230801

### CODE REVIEW

**Comment by hervyqa** [comment]
hi @ericonr any idea?

```
https://download.kde.org/stable/release-service/21.04.3/src/kamoso-21.04.3.tar.xz: Connection timed out
=> ERROR: kamoso-21.04.3_1: failed to fetch kamoso-21.04.3.tar.xz.
Error: Process completed with exit code 1.
```

File: srcpkgs/kamoso/template [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'kamoso'
+pkgname=kamoso
+version=21.04.3
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
```
Comment (Chocimier) [inline srcpkgs/kamoso/template:6]: We want to run tests.

File: srcpkgs/kamoso/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+hostmakedepends="
```
Comment (Chocimier) [inline srcpkgs/kamoso/template:7]: Add qt5-host-tools kcoreaddons kconfig for cross build.

File: srcpkgs/kamoso/template [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ qt5-quickcontrols gettext"
+makedepends="
+ glib-devel gst-plugins-base1-devel gst-plugins-bad1-devel gstreamer1-devel
+ kirigami2-devel knotifyconfig-devel purpose-devel qt5-multimedia-devel
+ qt5-quickcontrols2-devel"
+depends="
+ kconfig kconfigwidgets kcompletion kio ksolid knotifyconfig gst-plugins-good1
+ gst-plugins-bad1 phonon-qt5 qt5-graphicaleffects qt5-quickcontrols kirigami2
+ purpose"
+checkdepends="$depends"
```
Comment (Chocimier) [inline srcpkgs/kamoso/template:18]: makedepends pull needed packages, please remove checkdepends=

File: srcpkgs/kamoso/template [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+configure_args="-DBUILD_TESTING=OFF"
+hostmakedepends="
+ glib extra-cmake-modules kdoctools pkg-config boost python3 qt5-qmake
+ qt5-quickcontrols gettext"
+makedepends="
+ glib-devel gst-plugins-base1-devel gst-plugins-bad1-devel gstreamer1-devel
+ kirigami2-devel knotifyconfig-devel purpose-devel qt5-multimedia-devel
+ qt5-quickcontrols2-devel"
+depends="
+ kconfig kconfigwidgets kcompletion kio ksolid knotifyconfig gst-plugins-good1
```
Comment (Chocimier) [inline srcpkgs/kamoso/template:15]: kio, kconfig are autodetected, please remove.

**Review by Chocimier — COMMENTED** [review]
> any idea?

Network is unreliable universally. Please build locally or rebase and push again.

File: srcpkgs/kamoso/template (hunk 1 above)
Comment (hervyqa) [reply srcpkgs/kamoso/template]: ok. I try to fix. `configure_args="-DBUILD_TESTING=OFF"` i deleted it

File: srcpkgs/kamoso/template (hunk 2 above)
Comment (hervyqa) [reply srcpkgs/kamoso/template:6]: I have added `qt5-host-tools kcoreaddons kconfig`

File: srcpkgs/kamoso/template (hunk 3 above)
Comment (hervyqa) [reply srcpkgs/kamoso/template]: I have deleted `checkdepends="$depends"`

File: srcpkgs/kamoso/template (hunk 4 above)
Comment (hervyqa) [reply srcpkgs/kamoso/template]: I have deleted `kconfig kio`

**Comment by Johnnynator** [comment]
The commit message should be `New package: kamoso-21.04.3_1`

**Comment by hervyqa** [comment]
> The commit message should be `New package: kamoso-21.04.3_1`

done. `New package: kamoso-21.04.3_1`

**Pull request merged**